circleci-ruby/2.6.1-stretch-pgclient10.5-node10.13-browsers/Dockerfile

```diff
@@ -0,0 +1,175 @@
+FROM circleci/ruby:2.6.1-stretch
+
+USER root
+
+
+#### PG CLIENT
+
+ENV PG_VERSION 10
+
+RUN \
+  echo "deb http://apt.postgresql.org/pub/repos/apt/ stretch-pgdg main" > /etc/apt/sources.list.d/pgdg.list \
+  && curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+
+RUN apt-get update
+
+RUN apt-get install "postgresql-client-$PG_VERSION"
+
+
+#### NODE
+
+# Source: https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/ruby/images/2.5.1-stretch/node/Dockerfile
+
+## Using node installation from https://raw.githubusercontent.com/nodejs/docker-node/72dd945d29dee5afa73956ebc971bf3a472442f7/8/jessie/Dockerfile
+
+RUN groupadd --gid 1000 node \
+  && useradd --uid 1000 --gid node --shell /bin/bash --create-home node
+
+# gpg keys listed at https://github.com/nodejs/node#release-team
+RUN set -ex \
+  && for key in \
+  94AE36675C464D64BAFA68DD7434390BDBE9B9C5 \
+  FD3A5288F042B6850C66B31F09FE44734EB7990E \
+  71DCFD284A79C3B38668286BC97EC7A07EDE3FC1 \
+  DD8F2338BAE7501E3DD5AC78C273792F7D83545D \
+  C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
+  B9AE9905FFD7803F25714661B63B535A4C206CA9 \
+  56730D5401028683275BD23C23EFEFE93C4CFFFE \
+  77984A986EBC2AA786BC0F66B01FBB92821C587A \
+  8FCCA13FEF1D0C2E91008E09770F7A9A5AE15600 \
+  ; do \
+  gpg --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key" || \
+  gpg --keyserver hkp://ipv4.pool.sks-keyservers.net --recv-keys "$key" || \
+  gpg --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key" ; \
+  done
+
+ENV NODE_VERSION 10.13.0
```

Comment:
@usamah82 Node LTS is currently at `10.15.1` we can upgrade here.

Reply:
Will upgrade Node. Thanks for the info!